### PR TITLE
Upgrade to Guacamole 1.5.1

### LINKED ISSUE #37: Upgrade the version of the PostgreSQL Docker image
# 💡 Summary #

We currently pin to the PostgreSQL Docker image to `postgres:13` because of [a limitation in the JDBC PostgreSQL driver used in version `1.4.0` of the official Guacamole Docker images](https://github.com/cisagov/guacamole-composition/blob/cd27dbf66e8fe34780a663e0e54fa7aa34cb3803/docker-compose.yml#L51-L53).  It looks like [the JDBC PostgreSQL driver will be updated in the next release of the Guacamole Docker images](https://github.com/apache/guacamole-client/blob/master/guacamole-docker/bin/build-guacamole.sh#L111), at which point we should start using a newer version of the `postgres` Docker image.

## Motivation and context ##

A newer `postgres` Docker image is likely more performant and more secure.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Upgrades the Docker composition to use the recently-released  `guacamole/guacd:1.5.1` and `guacamole/guacamole:1.5.1` Docker images
- Upgrades the Docker composition to use a more recent version of the PostgreSQL Docker image.

## 💭 Motivation and context ##

Guacamole 1.5.1 is the latest release, and is therefore likely to be more performant and secure.

The ability to upgrade to a more recent version of the PostgreSQL Docker image is a side effect of the Guacamole Docker image upgrade.

Resolves #37.

## 🧪 Testing ##

I deployed these changes to our COOL staging environment and verified that they functioned as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.
- [x] Bump to version `1.0.0`.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

File: src/version.txt
Comment (dav3r) [inline src/version.txt]: I feel like we can probably move this repo to a version 1.0.0 at this point, don't you think?

**Comment by jsf9k** [comment]
We will need to wait for the release of version 1.5.1 because we are encountering [this bug](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1741?filter=allissues).  It appears that [version 1.5.1 should be released any day now](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1752?filter=allissues).

File: src/version.txt
Comment (jsf9k) [reply src/version.txt]: I will bump the version to `1.0.0` after I finalize the version upon approval.

**Comment by jsf9k** [comment]
> We will need to wait for the release of version 1.5.1 because we are encountering [this bug](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1741?filter=allissues). It appears that [version 1.5.1 should be released any day now](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1752?filter=allissues).

Version 1.5.1 has been released!

**Review by mcdonnnj — APPROVED** [review]
:shipit:

**Review by dav3r — APPROVED** [review]
🎉

File: src/version.txt
Comment (jsf9k) [reply src/version.txt]: It has been done.  @dav3r, could you resolve this conversation?

Comment (dav3r) [reply src/version.txt]: Resolved! 👍

**Pull request merged**
